### PR TITLE
Problem with the git+https installation fixed

### PR DESCRIPTION
**Pull request by gatocor**
Dear Jacob, 

I had quite a few problems installing your package from git as it did not install properly. 
I found it to be a small detail on how you arranged the package. I just put the code in a folder with the correct name and now it installs as expected with the git+https...

Best,
Gabriel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ how the projection indices are selected. It follows the
 
 Installation:
 ```
-pip install git+https://github.com/mossjacob/pcurvepy
+pip install git+https://github.com/gatocor/pcurvepy
```
Comment (mossjacob) [inline README.md]: this should go back to `mossjacob` for the readme in this repository

File: pcurve/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from pcurve.pcurve import *
+from pcurve.project_to_curve_converted import *
```
Comment (mossjacob) [inline pcurve/__init__.py]: `project_to_curve_converted` has been deleted in `master`, can you delete it here too? (It is not used anywhere).

**Review by mossjacob — CHANGES_REQUESTED** [review]
Thank you for your PR. Please see the comments before we merge

**Comment by gatocor** [comment]
I think that is it.

**Pull request merged**